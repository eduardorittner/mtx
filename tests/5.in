This is a test for
a simple deelete case
