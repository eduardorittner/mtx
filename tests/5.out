This is a test for
a simple delete case
